src/components/Navigation/tests/Navigation.test.tsx

```diff
@@ -39,11 +40,11 @@ describe('<Navigation />', () => {
     it('has a child with contentContext', () => {
       const Child: React.SFC<{}> = (_props) => {
         return (
-          <NavigationContext.Consumer>
+          <WithinContentContext.Consumer>
```

Comment:
We'll want to use `WithinContentContext` consumer rather than `NavigationContext` because it provides the `withinContentContext` value